tests/baselines/ArrowFunctionInsideTypeAssertion.baseline.txt

```diff
@@ -67,37 +67,37 @@ Grammar: TypeScript.tmLanguage
                    ^
                    source.ts meta.var.expr.ts meta.objectliteral.ts meta.object.member.ts cast.expr.ts meta.brace.angle.ts
                     ^
-                    source.ts meta.var.expr.ts meta.objectliteral.ts meta.object.member.ts cast.expr.ts meta.type.function.ts meta.parameters.ts punctuation.definition.parameters.begin.ts
+                    source.ts meta.var.expr.ts meta.objectliteral.ts meta.object.member.ts cast.expr.ts meta.type.cast.ts meta.type.function.ts meta.parameters.ts punctuation.definition.parameters.begin.ts
```

Comment:
As you can see there is already`cast.expr.ts` so this change isn't needed.

Reply:
`cast.expr.ts` includes the surrounding `<` and `>`, and i have `meta.type.cast.ts` include only the contents. (see diffs at https://github.com/microsoft/TypeScript-TmLanguage/pull/798/files#diff-d5222b162266b45fd998419db416fdddR1577 and https://github.com/microsoft/TypeScript-TmLanguage/pull/798/files#diff-d5222b162266b45fd998419db416fdddR1591

In my use case, i want to change the syntax highlighting of the contents of the cast, but not the surrounding punctuation (`<` `>`) or the `as`.

That was something i forgot to mention: since `cast.expr.ts` is used for `<MyType>variable` style-casts, should `variable as MyType` style casts also use `cast.expr.ts`, or is `meta.type.cast.ts` preferable? If they are switched to `cast.expr.ts`, should the `as` also have `cast.expr.ts`?

If i switch `variable as MyType` casts from `meta.type.cast.ts`, should the `as` also be scoped as `cast.expr.ts`, since

Reply:
The scenario you are looking for can be achieved by omitting `cast.expr.ts meta.brace.angle.ts`. We want to keep this simple and not add unnecessary scopes. Most themes do not care about the difference and hence we wont be accepting this change